lighthouse-core/test/runner-test.js

```diff
@@ -295,4 +296,96 @@ describe('Runner', () => {
       assert.strictEqual(AuditClass.meta.name, auditExpectedName);
     });
   });
+
+  it('results include artifacts when given artifacts and audits', () => {
+    const url = 'https://example.com';
+    const config = new Config({
+      audits: [
+        'is-on-https'
+      ],
+
+      artifacts: {
+        HTTPS: {
+          value: true
+        }
+      }
+    });
+
+    return Runner.run({}, {url, config}).then(results => {
+      if (!results.artifacts instanceof Object) {
```

Comment:
not sure if these `instanceof` checks are really necessary. If not an object, it will throw on the property accesses below anyway